src/main/kotlin/com/exactpro/th2/rptdataprovider/grpc/RptDataProviderGrpcHandler.kt

```diff
@@ -72,7 +72,7 @@ import java.util.concurrent.Executors
 import kotlin.coroutines.coroutineContext
 import kotlin.system.measureTimeMillis
 
-@EngineAPI
+//@EngineAPI

```

Comment:
This can be removed:

Official deprecation message
> This is not general purpose API and should be only used in custom server engine implementations.